modules/vulnerability-management20-configure-report-delivery-destinations-schedule.adoc

```diff
@@ -9,8 +9,23 @@
 You can optionally configure destinations for vulnerability reports and set up a schedule for delivery.
```

Comment:
I would maybe replace this with:

"Configuring destinations and delivery schedules for vulnerability reports is optional, unless you have included CVEs that were discovered since the last scheduled report. If you have included CVEs that were discovered since the last scheduled report, configuring destinations and delivery schedules for vulnerability reports is required."

Reply:
Good suggestion - changed slightly to reflect that it's an option they choose that makes these fields required.